testing/python-poetry/tests/nixpkgs_repositories.bzl

```diff
@@ -0,0 +1,53 @@
+load("@rules_nixpkgs_core//:nixpkgs.bzl", "nixpkgs_local_repository")
+load("@rules_nixpkgs_cc//:cc.bzl", "nixpkgs_cc_configure")
+load("@rules_nixpkgs_java//:java.bzl", "nixpkgs_java_configure")
+load("@rules_nixpkgs_python//:python.bzl", "nixpkgs_python_configure", "nixpkgs_python_repository")
+load("@rules_nixpkgs_core//:nixpkgs.bzl", "nixpkgs_package")
+
+def nixpkgs_repositories(*, bzlmod):
+    nixpkgs_local_repository(
+        name = "nixpkgs",
+        nix_file = "//:nixpkgs.nix",
+        nix_file_deps = ["//:flake.lock"],
+    )
+
+    # Tests implicitly depend on Java
+    nixpkgs_java_configure(
+        name = "nixpkgs_java_runtime",
+        attribute_path = "jdk11.home",
+        repository = "@nixpkgs",
+        toolchain = True,
+        register = not bzlmod,
+        toolchain_name = "nixpkgs_java",
+        toolchain_version = "11",
+    )
+
+    # Python depends on a CC toolchain being available
+    nixpkgs_cc_configure(
+        name = "nixpkgs_config_cc",
+        repository = "@nixpkgs",
+        register = not bzlmod,
+    )
+
+    nixpkgs_package(
+        name = "nixpkgs_python_configure_test",
+        nix_file = "//tests:python-test.nix",
+        repository = "@nixpkgs",
+    )
+
+    nixpkgs_python_configure(
+        python2_attribute_path = "python2",
+        repository = "@nixpkgs",
+        register = not bzlmod,
+    )
+
+    nixpkgs_python_repository(
+        name = "nix_pip",
+        repository = "@nixpkgs",
+        nix_file = "//:poetry.nix",
+        nix_file_deps = [
+            "//:pyproject.toml",
+            "//:poetry.lock",
+        ],
+    )
```

Comment:
Apart from this bit the file looks identical to `testing/python`. So, by that token it looks like they could be merged.